### PR TITLE
feat: worker run-result add static-file 

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

**Review by sguiheux — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16492.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: engine/worker/cmd_run_result.go [hunk 1]
```diff
@@ -32,9 +32,46 @@ func cdmAddRunResult() *cobra.Command {
...
+}
+
+func addStaticFileRunResultCmd() func(cmd *cobra.Command, args []string) {
+	return func(cmd *cobra.Command, args []string) {
+		if len(args) != 2 {
+			sdk.Exit("missing arguments. Cmd: worker run-result add static-file <name> <remote_url>")
+		}
+
+		name := args[0]
+		remoteURL := args[1]
```
Comment (fsamin) [inline engine/worker/cmd_run_result.go:64]: maybe we should check if it's a valid URL (url.Parse)

File: engine/worker/internal/handler_run_result.go [hunk 2]
```diff
@@ -36,58 +36,81 @@ func getRunResultHandler(ctx context.Context, wk *CurrentWorker) http.HandlerFun
 	}
 }
 
-func addRunResulthandler(ctx context.Context, wk *CurrentWorker) http.HandlerFunc {
+func addRunResultArtifactManagerHandler(ctx context.Context, wk *CurrentWorker) http.HandlerFunc {
```
Comment (fsamin) [inline engine/worker/internal/handler_run_result.go:39]: can you write a unit test of this handler (like we have a few ones)

**Review by fsamin — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16519.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by richardlt — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**